templates/app-autoscaler-deployment.yml

```diff
@@ -138,6 +138,8 @@ instance_groups:
         policy_db: *database
         cf:
           api: "https://api.((system_domain))"
+          client_id: ""
```

Comment:
why it is blank? I am assuming you will need that for API server given you have to interact with UAA

Reply:
yes, the `client_id ` and `secret` should not be blank, already fixed in the new commit